src/ShopBundle/objects/WebModules/ViewMyOrderDetails/adapter/TPkgShopViewMyOrderDetailsDbAdapter.php

```diff
@@ -45,6 +45,6 @@ public function getOrder($orderId)
     {
         $order = TdbShopOrder::GetNewInstance($orderId);
```

Comment:
We could simplify (in comprehensibility, not in lines of code) this method as follows:

- Call `GetNewInstance` without argument.
- Call `Load` with `$orderId`.
- Return null if `Load` returns `false`, else return the order.